src/main/java/org/fcrepo/camel/RdfNamespaces.java

```diff
@@ -32,6 +32,8 @@
 
     public static final String LDP = "http://www.w3.org/ns/ldp#";
 
+    public static final String PREMIS = "http://www.loc.gov/premis/rdf/v1#";
```

Comment:
The PREMIS namespace appears to only be used in tests. Is this addition needed or useful in `RdfNamespaces`?